test/framework/deployment_helpers.go

```diff
@@ -95,31 +95,76 @@ func DescribeFailedDeployment(input WaitForDeploymentsAvailableInput, deployment
 	return b.String()
 }
 
```

Comment:
Please move the input structs so each input struct is right above the corresponding func which takes it as a parameter